Jint/Runtime/Debugger/BreakPointCollection.cs

```diff
@@ -0,0 +1,99 @@
+﻿using Esprima;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Jint.Runtime.Debugger
+{
+    public class BreakPointCollection : ICollection<BreakPoint>
```

Comment:
ah now I see that it's a separte datatype, so `AddBreakPoint` not needed

Reply:
Yeah, the reason it "needs" to be at least a readable collection is that currently the only way to remove a breakpoint again is passing a reference to it - which e.g. in my case - means nasty LINQ `SingleOrDefault` shenanigans, rather than keeping my own list of the added breakpoints. Until `BreakPoint` has been looked through (equality etc.), I didn't see much idea in adding e.g. a `Remove(int line, int column, string source = null)` method. :-)